docs/reference/objects/blog_post.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Blog Post
+parent: Objects
+---
+
+When using a `blog_post` object you have access to the following attributes.
+
+# blog_post.author
+
+Returns the blog post's [author]({% link docs/reference/objects/author.md %}).
+
+# blog_post.body
+
+Returns the blog post's body as a string.
+
+# blog_post.hero_image
+
+Returns the blog post's hero [image]({% link docs/reference/objects/image.md %}).
+
+# blog_post.next_post
+
+Returns a `blog_post` object for the next post for the blog.
+
+# blog_post.previous_post
+
+Returns a `blog_post` object for the previous post for the blog.
+
+# blog_post.published_at
+
+Returns the date and time the blog post was published.
```

Comment:
How about "Returns the time the blog post was published as a timestamp.". Just to be clear it doesn't return two things.